models/location-sql.js

```diff
@@ -0,0 +1,58 @@
+import db from '../db/connection.js'
+
+// Get all location data
+export async function getAllLocationData() {
+    const result = await db.query(`SELECT * FROM location;`);
+    return result.rows;
+}
+
+// Get location data by loc_id
+export async function getLocationByLocID(loc_id) {
+    const result = await db.query(`SELECT * FROM location WHERE loc_id = $1;`, [loc_id])
+    return result.rows;
+}
+
+// Get location data by media_id
+export async function getLocationByMedID(media_id) {
+    const result = await db.query(`SELECT * FROM location WHERE media_id = $1;`, [media_id])
```

Comment:
I believe we will be recording the loc_id in the media table rather than the media_id in the location table